src/themes/admin_future/modules/contact/department.tpl

```diff
@@ -0,0 +1,80 @@
+            {foreach $DEPARTMENTS as $ROW}
+            {assign var="ARR_STATUS" value=[$LANG->getGlobal('disable'), $LANG->getGlobal('active'), $LANG->getModule('department_no_home')]}
+            <tr class="item" data-id="{$ROW.id}">
+                <td class="text-center align-middle" style="width:1%;">
+                    {if !empty($smarty.const.NV_IS_SPADMIN)}
+                    <select class="form-select department_cweight" data-default="{$ROW.weight}" style="width:fit-content">
+                        {for $WEIGHT = 1 to $COUNT}
+                        <option value="{$WEIGHT}" {if $WEIGHT == $ROW.weight}selected{/if}>{$WEIGHT}</option>
+                        {/for}
+                    </select>
+                    {else}
+                    {$ROW.weight}
+                    {/if}
+                </td>
+                <td class="align-middle full_name{if !empty($ROW.is_default)} is-default{/if}">
+                    <a href="#" class="department_view">{$ROW.full_name}</a>
+                </td>
+                <td class="align-middle text-nowrap" style="width:1%;">{$ROW.email}</td>
+                <td class="align-middle text-nowrap" style="width:1%;">{$ROW.phone|regex_replace:'/(\[|&#91;)[^\]]*(&#93;|\])$/':''}</td>
+                <td class="text-center align-middle">
+                    {if !empty($smarty.const.NV_IS_SPADMIN)}
+                    <select class="form-select department_cstatus" data-default="{$ROW.act}" style="width: 150px">
+                        {foreach $ARR_STATUS as $KEY => $STATUS}
+                            <option value="$KEY" {if $KEY == $ROW.act}selected{/if}>{$STATUS}</option>
+                        {/foreach}
+                    </select>
+                    {else}
+                    {$ARR_STATUS[$ROW.act]}
+                    {/if}
+                </td>
+                <td class="text-center align-middle" style="width:1%;">
+                    {if !empty($smarty.const.NV_IS_SPADMIN)}
+                    <input type="radio" name="is_default" class="form-check-input" value="{$ROW.id}" {if !empty($ROW.is_default)}checked{/if} aria-label="{$LANG->getModule('is_default_select')}">
+                    {elseif !empty($ROW.is_default)}
+                    <i class="fa-solid fa-check"></i>
+                    {/if}
+                </td>
+                {if !empty($smarty.const.NV_IS_SPADMIN)}
+                <td class="text-center align-middle text-nowrap" style="width:1%;">
+                    <button type="button" title="{$LANG->getGlobal('edit')}" aria-label="{$LANG->getGlobal('edit')}" class="btn btn-secondary btn-sm department_edit"><i class="fa-solid fa-pencil fa-lg"></i></button>
+                    <button type="button" title="{$LANG->getGlobal('delete')}" aria-label="{$LANG->getGlobal('delete')}" class="btn btn-secondary btn-sm department_del"><i class="fa-solid fa-trash fa-lg"></i></button>
+                </td>
```

Comment:
@danlt150 2 cái nút này icon bỏ fa-lg đi em nó thô lắm. Ngày trước font của mình có 12px cộng với thiết kế của bản 4 nên cái icon bé tí phải fa-lg cho nó to ra. Giờ font nó to ra và thiết kế của bản 6 nó to cái icon hơn, em để lg trên nền btn-sm nó to quá.

Cái nữa btn-sm ở này nên bỏ hoặc là cái select trạng thái và thứ tự thêm class `form-select-sm` để nó cao bằng nhau. Hiện tại 2 cái select thì to còn cái nút lại cho nhỏ hơn nhìn nó không cân đối lắm

Reply:
Thêm text-danger cho cái icon xoá hoặc btn-danger cho nút xoá nhé e.

Reply:
@danlt150 chưa thấy xử lý cái btn-sm em